deploy/kubernetes/driver/kubernetes/manifests/controller-server.yaml

```diff
@@ -110,6 +110,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: KUBE_NODE_NAME
```

Comment:
This changes need to be done
in following repos as well

1- satellite
2- operator
3- template
4- addon